src/aleph_client/commands/files.py

```diff
@@ -241,3 +244,43 @@ async def list(
                 typer.echo(f"Failed to retrieve files for address {address}. Status code: {response.status}")
     else:
         typer.echo("Error: Please provide either a private key, private key file, or an address.")
+
+
+class IpfsContent(BaseModel):
+    filename: Optional[str]
+    cid: str
+    url: str
+
+
+@app.command()
+async def ipfs_content(
+    item_hash: str = typer.Argument(..., help="Item hash of the store message"),
+    verbose: bool = True,
+) -> Optional[IpfsContent]:
+    """Return the underlying ipfs content for a given store message item hash"""
+
+    result, resp = None, None
+    async with AlephHttpClient(api_server=settings.API_HOST) as client:
+        try:
+            message, status = await client.get_message(item_hash=ItemHash(item_hash), with_status=True)
+            if status != MessageStatus.PROCESSED:
+                resp = f"Invalid message status: {status}"
```

Comment:
if I understand correctly, the difference between this new command and the existing `download` command is that the download take the ipfs hash and this one take the aleph item_hash?

Reply:
I needed a method to retrieve the IPFS CID from a store message and it seems to be a good idea to add this command
```bash
❯ aleph file ipfs-content 2f50b6d078005074801260bac1f8860b10d2a92cf00c91459800ea6042a02cc9
Filename: None
CID: QmYJr5P4CeJBfVETuSn62gbzFZpSiTYkTegdGuQFu7QsVn
URL: https://ipfs.aleph.cloud/ipfs/QmYJr5P4CeJBfVETuSn62gbzFZpSiTYkTegdGuQFu7QsVn
```
Along current commands:
![image](https://github.com/user-attachments/assets/4c40f034-2f10-4716-9c54-46645e9ccc47)

Reply:
I will suggest modifying that current `download` command to be able to use it in both cases, like knowing the IPFS `item_hash` or the storage `item_hash`, and also download the underlying file using the `STORE` message `item_hash`. Put that option on the SDK directly will be better also, to avoid putting logic on the CLI side.